source/includes/_queries.md

```diff
@@ -347,21 +396,17 @@ message AccountAssetResponse {
 
 Field | Description | Constraint
 -------------- | -------------- | --------------
-Asset ID | asset id to knows its balance | `[a-z]{1,9}\#[a-z]{1,9}`
-Account ID | account which has this balance  | `[a-z]{1,9}\@[a-z]{1,9}`
+Asset ID | asset id to knows its balance | `[A-Za-z0-9]{1,9}\#[A-Za-z0-9]{1,9}`
+Account ID | account which has this balance  | username@domain
 Balance | balance of asset | > 0
 
-### Validation
-
-Stateless validation (check timestamp and signature)
-
 ## Get asset info
 
 ### Purpose
 
-In order to know precision for given asset, and other related info in future, such as description, etc. user can send `GetAssetInfo` query to Iroha network
+In order to know precision for given asset, and other related info in future, such as description of asset, etc. user can send `GetAssetInfo` query to Iroha network
```

Comment:
I think "to Iroha network", is unnecessary, as a reader is unlikely to confuse that with some other blockchain network.